cmd/controller/controller-runtime-main.go

```diff
@@ -42,12 +45,24 @@ type SchemeFunc func(*runtime.Scheme) error
 // ProvideFunc adds a controller to a Manager.
 type ProvideFunc func(manager.Manager) (controller.Controller, error)
 
+var knownProvideControllers = map[string]ProvideFunc{
+	"eventtype.feeds.knative.dev":       eventtype.ProvideController,
+	"feed.feeds.knative.dev":            feed.ProvideController,
+	"flow.flows.knative.dev":            flow.ProvideController,
+	"subscription.eventing.knative.dev": subscription.ProvideController,
+}
+
 // controllerRuntimeStart runs controllers written for controller-runtime. It's
 // intended to be called from main(). Any controllers migrated to use
 // controller-runtime should move their initialization to this function.
-func controllerRuntimeStart() error {
+func controllerRuntimeStart(logger *zap.SugaredLogger) error {
 	logf.SetLogger(logf.ZapLogger(false))
 
+	cm, err := configmap.Load("/etc/config-controllers")
```

Comment:
Can this line have a comment explaining the purpose and intended behavior? For example, does an update to the configmap also change the set of running controllers inside an existing controller manager pod?